src/generator/AutoRest.Go/Templates/VersionTemplate.cshtml

```diff
@@ -24,8 +24,8 @@ import (
 @EmptyLine
 
 const (
-    major = "3"
-    minor = "1"
+    major = "4"
```

Comment:
I'd prefer the version number be coming from the AutoRest options (PackageVersion) instead of hard coding it and updating it every release.